com.unity.formats.alembic/Documentation~/ref_Importer.md

```diff
@@ -74,6 +75,16 @@ This section of the Materials tab lists all meshes and submeshes of the imported
 | (2) | Submeshes of mesh (1), identified by the names of their corresponding Face Sets in the original Alembic file.<br /><br />**Note:** "Empty" means the corresponding Face Set has no name in the original Alembic file. In that case, you can only assign a material manually. |
 | (3) | The current material override status of the submesh.<br /><br />Use the target fields to manually assign materials to the listed submeshes. For automatic name-based material assignation, use [Material Search](#material-search).<br /><br />**Note:** "None (Material)" means the Alembic Importer still assigns the Default Material to the submesh. |
 
+## Hair
+
+The Hair tab helps you generate a groom from the imported Alembic file.
+
+![The Import Settings window (Hair tab)](images/abc_import_options_hair.png)
```

Comment:
There are two others screenshots in this page showing the Model/Material/Hair tabs. They should be also be updated I believe.

Reply:
yes this is true! However, I discussed this with @sebastienduverne and the screenshots system will be getting overhauled soon so it's not vital to replace these screenshots yet. The sample information in the current screenshots have is meaningful and therefore it's better to keep these rather than having new images with all 3 tabs

Reply:
Yeah I remember I said that – however for now there is no immediate plan to retake all screenshots across the doc (for context, it would be to switch to dark mode).
So if one of you could update the 3 consistently (reproduce what was shown as best as possible) and uniformly (get the same width and resolution for all), that would be cleaner. I can't really do it at the moment, I'm not set up for it.
Thanks!